prepareShapesAndCards.py

```diff
@@ -623,6 +663,11 @@ def prepareShapes(backgrounds, signals, discriminant, discriminantName):
             bbb.SetAddThreshold(0.1)
             bbb.AddBinByBin(cb.cp().backgrounds(), cb)
             bbb.AddBinByBin(cb.cp().signals(), cb)
+        else:
```

Comment:
I think it is worth having a simple way in the options to remove all bbb, also for QCD. Here setting nobbb will still leave QCD bbb which is not a nice behavior IMHO

Reply:
But in that case the presented signal strength ('r') may vary by a large fraction which could lead to misunderstanding of our result. For example, 2017 Hct, -0.31 -> 0.12 (this is old number but enough to show difference)